package.json

```diff
@@ -42,7 +42,7 @@
   "main": "build/electron.js",
   "homepage": "./",
   "build": {
-    "productName": "Elastic Synthetics Recorder",
+    "productName": "Elastic-Synthetics-Recorder",
```

Comment:
Align the existing approach done in the releases, see https://github.com/elastic/synthetics-recorder/releases/tag/untagged-f5efbb08222e46cedb8f